packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -52,7 +53,23 @@ export function _publishCloudEventSend(
       contentType:
         (options.contentType as any) ??
         "application/cloudevents+json; charset=utf-8",
-      body: { event: event },
+      body: {
+        event: {
+          id: event["id"],
+          source: event["source"],
+          data: event["data"],
+          data_base64: uint8ArrayToString(
+            event["dataBase64"] ?? new Uint8Array(),
+            "base64"
+          ),
+          type: event["type"],
+          time: new Date(event["time"] ?? ""),
```

Comment:
I know the `new Date` is not introduced in this pr, but personally I would prefer not to init the optional date with `new Date()`. This would mislead the customers because we give them a default value not undefined.

@joheredi How do you think?

Reply:
resolved.